cvmfs/statistics.h

```diff
@@ -45,6 +45,13 @@ class Counter {
   atomic_int64 counter_;
 };
 
+// perf::Func(Counter) is more clear to read in the code
+inline void Dec(class Counter *counter) { counter->Dec(); }
```

Comment:
With counter++ you'd need to think if this is atomic or not when you see it in the code.  `Inc()` or `Dec()` can be immediately recognized.